pennylane/_device.py

```diff
@@ -16,10 +16,11 @@
 """
 # pylint: disable=too-many-format-args
 import abc
+from collections.abc import Iterable
```

Comment:
Thanks for updating this, it has been on the back of my mind for a while that this is deprecated as of Python 3.9!